src/flask_container_scaffold/app_configurator.py

```diff
@@ -0,0 +1,76 @@
+from flask_container_scaffold.util import load_yaml, load_cfg
+
+class AppConfigurator(object):
```

Comment:
I think there may be an issue with the flake8 job, there should be two empty lines before the class, and only one between the methods in the class. Actually looking at the job it looks like we're skipping the source directory altogether and only scanning the tests & setup.py file!! We need to add src to the list.